src/pages/Profile/Profile.tsx

```diff
@@ -63,7 +100,23 @@ const Profile: FC = () => {
     )
   }
 
+  function renderEditButton() {
+    if (!isCurrentUser) return null
+
+    return <Chip className={classes.editButton} color="primary" label="Edit Info" />
+  }
+
+  function getUserRoleString(role: number) {
+    if (role === UserRole.Mid) return 'Mid'
+    if (role === UserRole.Senior) return 'Senior'
+    if (role === UserRole.TeamLead) return 'TeamLead'
+    if (role === UserRole.God) return 'God'
+    return 'Junior'
+  }
```

Comment:
How about using switch? 
```suggestion
  function getUserRoleString(role: number) {
  switch (role) {
    case UserRole.Mid:
        return 'Mid'
     case UserRole.Senior:
        return 'Senior'
     case UserRole.TeamLead:
        return 'TeamLead'
     case UserRole.God:
        return 'God'
      default:
        return 'Junior'
  }
  }
```